src/main/kotlin/logic/usecase/task/GetTasksByProjectUseCase.kt

```diff
@@ -0,0 +1,9 @@
+package org.damascus.logic.usecase.task
+
+import logic.model.Task
+import logic.repo.TaskRepository
+import java.util.*
+
+class GetTasksByProjectUseCase (private val repository: TaskRepository) {
```

Comment:
Name can be improved to `GetProjectTasksUseCase` or `GetTasksByProjectIdUseCase`.